html-aam/index.html

```diff
@@ -2251,38 +2251,45 @@ <h4 id="el-figcaption">`figcaption`</h4>
                 <a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a>
               </th>
               <td>
-                <div class="role"><span class="type">Roles:</span> `ROLE_SYSTEM_TEXT`; `IA2_ROLE_CAPTION`</div>
-                <div class="relations"><span class="type">Relations:</span> `IA2_RELATION_LABEL_FOR` with parent <a href="#el-figure">`figure`</a> element</div>
+                <div class="role"><span class="type">Role:</span> Use WAI-ARIA mapping</div>
+                <div class="relations">
+                  <span class="type">Relations:</span>
+                  If the first instance of a child `figcaption` to its parent <a href="#el-figure">`figure`</a>, then `IA2_RELATION_DETAILS_FOR` with parent `figure`.
+                </div>
               </td>
             </tr>
             <tr>
               <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
               <td>
-                <div class="ctrltype"><span class="type">Control Type:</span> `Text`</div>
+                <div class="ctrltype"><span class="type">Role:</span> Use WAI-ARIA mapping</div>
               </td>
             </tr>
             <tr>
               <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
               <td>
-                <div class="role"><span class="type">Role:</span> `ATK_ROLE_CAPTION`</div>
+                <div class="role"><span class="type">Role:</span> Use WAI-ARIA mapping</div>
                 <div class="relations">
                   <span class="type">Relations:</span>
-                  `ATK_RELATION_LABEL_FOR` with parent <a href="#el-figure">`figure`</a> element
+                  If this is the first instance of a child `figcaption` to its parent <a href="#el-figure">`figure`</a> element, then expose `RELATION_DETAILS_FOR` referencing the parent `figure`
```

Comment:
@scottaohara @jcsteh 
JAWS turned off their details relation setting by default because it made some pages overly verbose. Is it really necessary when the figcaption is most likely just a sibling of the image being described anyway? Should we file a separate issue for this to remove the automatic assignment of the details relation for it? I don't think I understand why we need it.

Reply:
i'm not opposed to removing the details relationship.  i think you're right here.  i think what's most important here is to make sure it's clear what the figure content is vs the caption.  the divide of where each start/end.  but the details relationship isn't needed for that.

Reply:
if people are good with removing this, then i can pull all that out of this PR.  it'd definitely make it a lot simpler.

Reply:
If we remove the details relationship, am I correct that the only thing this PR will do is remove the label association between figure and figcaption?

> i think what's most important here is to make sure it's clear what the figure content is vs the caption. the divide of where each start/end.

How will this PR achieve that? Or am I misunderstanding something here?

Reply:
@jcsteh right now if we don't do anything else but remove the detail relationship from the PR, then yes, this PR would simply be about removing figcaption from contributing to the accName of the figure.

per my comment on making the distinction between the figure and its caption clear, i'm open to ideas if the existing caption role for the figcaption can't be the delineator that screen readers could use to identify this separation.

Reply:
Personally, I think the role is a sufficient delineator.